dyn_em/module_first_rk_step_part1.F

```diff
@@ -34,6 +34,7 @@ SUBROUTINE first_rk_step_part1 (   grid , config_flags              &
                              , f_flux                           &
                              , aerocu                           &
                              , restart_flag                     &
+                             , feedback_is_ready                &
```

Comment:
@dwongepa 
David,
With the NMM building failing, this argument could be the cause. Perhaps it can be optional, and then used in that fashion?